example/test_driver/app_test.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter_driver/flutter_driver.dart';
+import 'package:test/test.dart';
+
+void main() {
+  final selectVideoBtnFinder = find.byValueKey('select_video');
+  final clearCacheBtnFinder = find.byValueKey('clear_cache');
+  final statusFinder = find.byValueKey('status');
+  final outputFileSizeFinder = find.byValueKey('output_file_size');
+
+  FlutterDriver driver;
+
+  setUpAll(() async {
+    driver = await FlutterDriver.connect();
+  });
+
+  tearDownAll(() async {
+    if (driver != null) {
+      driver.close();
+    }
+  });
+
+  test('compress video and clear cache', () async {
+    expect(await driver.getText(statusFinder), 'init');
+
+    // Initialize compression
+    await driver.tap(selectVideoBtnFinder);
+
+    // Wait for compression to finish
+    await driver.waitFor(find.text('compressed'));
+
+    // Output file should not be empty
+    final outputFileSize = await driver.getText(outputFileSizeFinder);
+    expect(int.parse(outputFileSize), greaterThan(0));
```

Comment:
Expect `compressedSize > 0 && compressedSize < originalFileSize`

Reply:
Not necessarily

In this case, the original file size of the sample is so small (since it's just text on white bg), the compressed version is actually bigger.

Reply:
then I would rather put some real video, just to make sure it compress

Reply:
I've updated the video and the test now